code/modules/tram/tram.dm

```diff
@@ -0,0 +1,227 @@
+/obj/tram/ex_act(severity)
+	switch(severity)
+		if(1.0)
+			del(src)
+			return
+		if(2.0)
+			if(prob(50))
+				del(src)
+			return
+		if(3.0)
+			if(prob(25))
+				del(src)
+			return
+
+/obj/tram/blob_act()
+	if(prob(50))
+		del(src)
```

Comment:
@Iamgoofball people have this thing called 'life' and 'being busy' and if you followed the community you'd know both of those mythical, rare events occurred for Glloyd. Update is scheduled this week, IIRC, as a matter of fact.